src/sass/_shame.scss

```diff
@@ -201,12 +201,30 @@ body {
     }
 
     .usa-width-three-fourths {
-      margin-right: 2.35765%;
       width: 74.41059%;
     }
   }
 }
 
+// USWDS override so that checkboxes aren't offscreen
+// At some point between 0.10 and 1.4, USWDS modified radio/checkbox style
+// so be position: absolute, and left: -999em. This isn't compatible with our
+// e2e test cases, because Nightwatch can't find the elements on screen.
+input[type="checkbox"], input[type="radio"] {
+  margin-left: -2rem;
+  opacity: 0;
+  position: absolute;
+  left: auto;
+}
+
+// USWDS uses this:
+// [hidden] { display: none !important; }
+// However, some apps set the property to false and expect the element to be visible.
+// So, we do this shameful override to get around it.
+[hidden=false]{
+  display: block !important;
```

Comment:
To avoid shameful overrides, will this be resolved in a later PR where you do `removeAttribute` instead of setting `hidden` to `false`?

Reply:
That was my first instinct (there's a revert commit on here where I was doing that previously). The file that toggles the `hidden` property is being removed though, and I don't know what it's doing instead, so I figured this was the safer route. But depending on what that file looks like we should be able to remove this rule.